scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/SparkeyUri.scala

```diff
@@ -88,6 +88,8 @@ private[sparkey] class SparkeyWriter(val uri: SparkeyUri) {
 
   def close(): Unit = {
     delegate.flush()
+    // TODO: configure how much memory to use for hash table
+    // delegate.setMaxMemory(max memory usage in bytes)
```

Comment:
is it ok if we move this comment to an issue? easier to track and increases chances of someone picking it up.

Reply:
We can merge now and file ticket to track.